Assignments/stu1/assignments.py

```diff
@@ -186,9 +186,9 @@ def save_to_cat_table(cat):
             with conn.cursor() as cur:
                 cur.execute(INSERT_CAT, (cat['cat_id'], cat['cat_name'], cat['status']))
 
-def get_cat(cat_id):
+def get_cat(cat):
```

Comment:
No.  The `get_cat` function is supposed to return a cat from the database for the specified cat id passed into this function.  The select statement needs to change, and we need to pass in the id of the cat into this function.